gobdd.go

```diff
@@ -642,6 +668,16 @@ func (s *Suite) findStepDef(text string) (stepDef, error) {
 	return sd, nil
 }
 
+func (s *Suite) skipContainer(containerTags []*msgs.Tag) bool {
```

Comment:
this name is confusing. I'd call this function `shouldSkipStep` or `shouldIgnoreStep`. WDYT?

Reply:
Changed the name to be more explicit.